README.md

```diff
@@ -49,17 +50,13 @@ These reside in the [integration features directory](features/integration)
 | ---------------------- | -------------------------------------------------------------------------------------- |
 | @abi                   | Test the Application Binary Interface (ABI) with atomic txn composition and execution. |
 | @algod                 | General tests against algod REST endpoints.                                            |
-| @application.evaldelta | Test that eval delta fields are included in algod and indexer.                         |
```

Comment:
Propose the removal of the `evaldelta` feature as it hasn't been running anywhere since February 2022.

Reply:
@winder @jasonpaulos ^^^

Reply:
I believe Indexer has proper unit tests to verify the encoding for these fields now. So removing it is fine by me.